.travis.yml

```diff
@@ -53,8 +53,9 @@ jobs:
     - find build-support -name "*.py[co]" -delete
     - ./build-support/bin/prune_travis_cache.sh
     before_install:
+    - pyenv global
     - ./build-support/bin/install_aws_cli_for_ci.sh
-    - pyenv global 2.7.15 3.6.7 3.7.1
+    - pyenv global 2.7.15 3.6.7 3.7.1 3.8.0
```

Comment:
Should this be locked on 3.8.0 ? The latest I think it's 3.8.6, shouldn't we be using at least 3.8.4~5 ? 
Same for the other minor version on 3.6 and 3.7.

Reply:
I'm iterating to see what Travis has pre-installed in the Ubuntu Xenial environment. I'm not even sure if they have 3.8.0. If they don't, I'll need to land a precursor PR that upgrades us from Xenial to Bionic.